### ISSUE TITLE
Built binaries do not include or find VC Redist binaries

### ISSUE THREAD

**Issue by wojciechka** [ojdkbuild/ojdkbuild#59]
When trying to run Tomcat and Elasticsearch with this project, I've noticed ojdkbuild binaries for Windows x64 do not include any of the VC Redist and Windows SDK redist libraries.

I am copying `jre` folder from `ojdkbuild` along with Tomcat and Elasticsearch apps, each going into their directory structure.

When trying to copy the dependent binaries into `jre/bin` folder, the binaries are not picking it up either. They are looked up based on current working directory, but not where JRE/JDK is installed.

This is important when running a more complex deployment that requires working directory to be set to that application's path and shipping multiple Java based applications.

The only workaround I was able to find is to copy the required libraries into both Tomcat and Elasticsearch folders that are working directories when the application is started, but ideally this build should be able to locate the libraries in `jre/bin` folder (i.e. parent folder of `server/jvm.dll`) on its own, same as Oracle builds have been doing in the past.

This or related item is described in more details in an OpenJDK project issue:

https://bugs.openjdk.java.net/browse/JDK-8202557?focusedCommentId=14176391&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-14176391

**Comment by ojdkbuild** [comment]
Hi, some info on VC redist libs:

1. jdk8 is built with VC2010, it includes `msvcr100.dll` in `bin` and `jre/bin` directories

2. jdk11 is built with VC2017, redist libs in it are split into multiple files with prefixes `api-ms-win-core-` and `api-ms-win-crt-`, these files are included inside `bin` directory

3. OpenJFX 8 (that is included with jdk8 MSI) is built with VC2017, it includes the set of `api-ms-win-core-` and `api-ms-win-crt-` libs in addition to `msvcr100.dll`

4. jdk8 is likely going to be updated to use VC2017 in one of the following releases (probably `8u211`), after that it is going to include the set of `api-ms-win-core-` and `api-ms-win-crt-` inside both `bin` and `jre\bin` directories.


On running Tomcat with `procrun/prunsrv` custom launcher with Windows SCM integration: please see [this issue](https://github.com/ojdkbuild/ojdkbuild/issues/34#issuecomment-423423370) for details.

In short: jdk8 up to `u181` used separate shared `zlib` library (to stay closer to RHEL). It caused problems with `procrun` and `8u191` includes `zlib` linked as a static library (how it is done by other jdk-windows builds).

In `8u191` another `procrun` issue [was reported](https://github.com/ojdkbuild/ojdkbuild/issues/34#issuecomment-442150503) about the incorrect set of dependencies for `jvm.dll`. It was harmless for `java.exe/javaw.exe` launchers, but appeared to be a problem for `procrun`. This was fixed and going to be included into `8u201`. You can try and [alternate 8u191 build](https://github.com/ojdkbuild/ojdkbuild/issues/50#issuecomment-439705721) that includes these changes.

**Comment by wojciechka** [comment]
Thank you. Do you have any plans for releasing `8u201` soon? I've seen some commits, but it is not released yet.

I'd be more than happy to test it and confirm whether it worked.

**Comment by ojdkbuild** [comment]
Yes, `8u201` is almost ready and should be published in some hours.

**Comment by wojciechka** [comment]
Thank you, I will verify it early next week.

Do you happen to know regarding plans to also update Linux binaries at https://github.com/ojdkbuild/contrib_jdk8u-ci/releases ?

**Comment by ojdkbuild** [comment]
Thanks for reminder about Linux builds - building it now, should appear in some minutes.

**Comment by wojciechka** [comment]
Thank you for all of this.

I also confirm that the original issue is now resolved, so I'll close this issue.
